searx/engines/wallhaven.py

```diff
@@ -76,7 +76,7 @@ def response(resp):
                 'url': result['url'],
                 'img_src': result['path'],
                 'thumbnail_src': result['thumbs']['small'],
-                'img_format': result['resolution'],
+                'resolution': result['resolution'].replace('x', ' x '),
```

Comment:
Wallhaven returns something like `1920x1080`as image resolution, while for other engines, we have a gap in between the numbers and the `x`, like `1920 x 1080`. So this change is just for consistency, although I don't think anyone would actually notice the difference really :)